api/google-search/src/main/java/com/google/search/googlesearch/models/SearchAutoCompleteModel.java

```diff
@@ -0,0 +1,23 @@
+package com.google.search.googlesearch.models;
+
+import java.util.ArrayList;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(value = "prototype")
+public class SearchAutoCompleteModel {
+	private ArrayList<String> results = new ArrayList<String>();
```

Comment:
Please follow the rule "code to Interface not to implementation"  and avoid instantiating global variable combing with @Component annotation otherwise it will give unexpected results in multithreading environment.

Reply:
Removed Global class member variable initialization and removed @Component annotation